src/lib/holocene/pagination.svelte

```diff
@@ -76,9 +77,42 @@
   };
 
   $: floatStyle = getFloatStyle({ width, height, screenWidth });
+
+  async function handleKeydown(event: KeyboardEvent) {
```

Comment:
One odd edge case is if you select the dropdown for page size, select a size, and then attempt to use the row shortcuts it goes up/down a row and then opens up the dropdown again 🔽 

![Screenshot 2023-01-12 at 1 10 58 PM](https://user-images.githubusercontent.com/15069288/212170968-569a252c-4b95-44f0-abd7-33783e900be1.png)

Reply:
Good find! That is weird